wiki/models/custom-sabers.md

```diff
@@ -26,7 +26,7 @@ found [here](https://twitter.com/nalulululuna/status/1493263219485405184).
 
 ### Quest Sabers
 Custom Sabers for Quest are provided by the Qosmetics mod. The latest version can be found in the
-[#quest-mods channel](https://discord.gg/beatsabermods) in the BSMG Discord. Sabers are called `qsabers` and are currently
+[#quest-mods channel](https://discord.gg/beatsabermods) in the BSMG Discord. Sabers are called `whackers` and are currently
```

Comment:
Should add a mention on the legacy qsaber files and have a short summary on what the difference between them are.